package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "skipto",
   "description": "This plugin provides a dynamically-generated drop-down menu that allows keyboard and screen reader users to quickly skip to the most important places on the webpage.",
-  "version": "4.0.1",
+  "version": "3.1.5",
```

Comment:
```suggestion
  "version": "4.0.2",
```

Reply:
Why version 4.0.2?

Reply:
Because the last changes you made were a backwards compatibility break the version number was bumped to 4.0.1. So you will need to make this change and then recompile.

Reply:
This sounds like a good idea, but I need to coordinate with some other people at Illinois.  It is a three day weekend here , so it maybe a few days before I can get back to you on this change.   

Thank you for you interest n the updated version of SKipTo.

Reply:
accepted and files updated